src/main/kotlin/george/backend/hiring/KotlinApplication.kt

```diff
@@ -0,0 +1,5 @@
+package george.backend.hiring
+
+fun main(args: Array<String>) {
+    println("Hello, World!")
```

Comment:
**[CR]** The class intentionally had no `main`
Reasons:
- we had candidates who did not know how to write a `main` even with google
- for some candidates having two `main` methods confused them and their IDE setups